File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var runningSum = function(nums) {
    let arr=[];
    let x=nums[0];
    arr.push(x)
    for(let i=1;i<nums.length;i++){
        x+=nums[i];
        arr.push(x);
    }
    return arr
};

File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.cpp
class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        int length=nums.size();
        int sum=nums[0];
        for(int i=1;i<length;i++){
            sum+=nums[i];
            nums[i]=sum;
        }
        return nums;
    }
};